front/app/hooks/useTotalSeats.ts

```diff
@@ -0,0 +1,30 @@
+import useAppConfiguration from 'api/app_configuration/useAppConfiguration';
+import { isNil } from 'utils/helperUtils';
+import useFeatureFlag from './useFeatureFlag';
+
+export default function useTotalSeats() {
+  const { data: appConfiguration } = useAppConfiguration();
+  const seatBasedBillingEnabled = useFeatureFlag({
+    name: 'seat_based_billing',
+  });
+
+  if (isNil(appConfiguration)) return null;
+
+  const core = appConfiguration.data.attributes.settings.core;
+
+  const additionalAdmins = seatBasedBillingEnabled
```

Comment:
🤔 Minor and we could probably ignore. I'll leave to you. 
```
  const additionalAdmins = seatBasedBillingEnabled && core.additional_admins_number ?? 0;
```

Reply:
For me, the ternary operator is a bit clearer. The rest is fixed.